src/Otp.php

```diff
@@ -91,9 +91,20 @@ public function totp($secret, $timecounter = null)
     /* (non-PHPdoc)
      * @see Otp.OtpInterface::checkHotp()
     */
-    public function checkHotp($secret, $counter, $key)
+    public function checkHotp($secret, $counter, $key, $counterwindow = 5)
```

Comment:
This is a checking function call, I hope you wouldn't use the output of it directly as the output for the API response :) That's why mixing values is often not a good idea, but here it's a function whose output is then used for something else in the logic of the caller before returned anywhere externally. In either case, you want something in between to do the checks for replay attacks.